Signal/src/call/NonCallKitCallUIAdaptee.swift

```diff
@@ -27,12 +27,10 @@ class NonCallKitCallUIAdaptee: CallUIAdaptee {
 
         let call = SignalCall.outgoingCall(localId: UUID(), remotePhoneNumber: handle)
 
-        CallService.signalingQueue.async {
-            _ = self.callService.handleOutgoingCall(call).then {
-                Logger.debug("\(self.TAG) handleOutgoingCall succeeded")
+        self.callService.handleOutgoingCall(call).then {
```

Comment:
probably gettting a warning here without explicitly ignoring the returned value, right?

i.e. put back the `_ = `